src/main/scala/BintrayKeys.scala

```diff
@@ -0,0 +1,84 @@
+package bintray
+
+import sbt._
+
+trait BintrayKeys {
+  val bintray = taskKey[String]("bintray-sbt is an interface for the bintray package service")
+
+  val bintrayRelease = taskKey[Unit](
+    "Releases a version of package on bintray")
+
+  val bintrayReleaseOnPublish = settingKey[Boolean](
+    "When set to true, publish also runs bintrayRelease.")
+
+  val bintrayOrganization = settingKey[Option[String]](
+    "Bintray organization name to publish to. Defaults to None unless project is an sbtPlugin")
+
+  val bintrayPackage = settingKey[String](
+    "Bintray package name")
+
+  val bintrayRepository = settingKey[String](
```

Comment:
Yes. sbt 0.13 recommendation is to use the lowercase `settingKey` and camelCase for keys. Without prefix name collision will happen at on the kv list, so the logical outcome is that both Scala identifier and key name gets prefixed.
I had to make similar change for sbt-assembly as well - https://github.com/sbt/sbt-assembly/blob/9416423a5c16c1a1ef3bf36b7c3286fd69c35bff/src/main/scala/sbtassembly/AssemblyKeys.scala